string character
---
geometry_msgs/Point[] points
